packages/editor/src/client/panel/teaserSelectPanel.tsx

```diff
@@ -230,6 +236,17 @@ export function TeaserSelectPanel({onClose, onSelect}: TeaserSelectPanelProps) {
       case TeaserType.PeerArticle:
         return (
           <>
+            {loadingPeerArticles && (
+              <div>
+                <hr />
+                <hr />
+                <Loader
+                  backdrop
+                  size="md"
+                  content={t('articleEditor.panels.loadingPeerArticles')}
```

Comment:
Do we have this translation in the translation files already available?

Reply:
??? Isn't that the one that is in the pr? or what do you mean?